bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -2308,28 +2331,4 @@ public static Image win32_new(Device device, int type, long handle) {
 	image.handle = handle;
 	return image;
 }
-
-/**
- * Invokes platform specific functionality to adapt an image for
- * the correct zoom.
- * <p>
- * <b>IMPORTANT:</b> This method is <em>not</em> part of the public
- * API for <code>Image</code>. It is marked public only so that it
- * can be shared within the packages provided by SWT. It is not
- * available on all platforms, and should never be called from
- * application code.
- * </p>
- *
- * @param image the image to adapt for the provided zoom
- * @param targetZoom zoom in % of the standard resolution
- * @return an image object equal to the specified image scaled to the provided targetZoom
- *
- * @noreference This method is not intended to be referenced by clients.
- */
-public static Image win32_new(Image image, int targetZoom) {
-	if (targetZoom != image.getZoom()) {
-		image.handleDPIChange(targetZoom);
-	}
-	return image;
-}
-}
+}
```

Comment:
Please remove that change on the last line

Reply:
Done